assets/stylesheets/gu-sass/colours.scss

```diff
@@ -99,6 +99,7 @@ $gu-colours: (
   news-garnett-feature: #ad0006, //used for feature head
   news-garnett-media-main-1: #ff4e36, // used for kicker on media
   news-garnett-highlight: #ffe500,
+  news-garnett-highlight-hover: #fff000,
```

Comment:
@Amohkhan are we able to use something from the palette here instead? Or a darken/lighten variant?